index.js

```diff
@@ -339,7 +340,8 @@ function joinRoom (room) {
         let doc = node.value
         if (doc.username && doc.publicKey) {
           usernames[doc.publicKey] = doc.username
-          $(`#a${doc.publicKey} div.person-name`).text(doc.username)
+          $(`#a${doc.publicKey} .person-name`).text(doc.username)
+          $(`#a${doc.publicKey} .avatar`).attr('src', doc.image)
```

Comment:
There's a case where this fires after the `stream` event which is why we store it. You'll want to do something similar down in the `stream` event handler as you do here to set the image.